Configuration/python/processingUtilities.py

```diff
@@ -979,6 +990,34 @@ def customizeMINIAODElectronVID(process, collections, usedCollections):
 
     return process
 
+def customizeMINIAODPhotonVID(process, collections, usedCollections):
+    if not hasattr (collections, "photons"):
+        return process
+
+    from PhysicsTools.SelectorUtils.tools.vid_id_tools import DataFormat,switchOnVIDPhotonIdProducer,setupAllVIDIdsInModule,setupVIDPhotonSelection
+    switchOnVIDPhotonIdProducer(process, DataFormat.MiniAOD)
+
+    my_id_modules = ['RecoEgamma.PhotonIdentification.Identification.cutBasedPhotonID_Spring16_V2p2_cff']
+
+    if os.environ["CMSSW_VERSION"].startswith ("CMSSW_9_4_"):
+        my_id_modules = ['RecoEgamma.PhotonIdentification.Identification.cutBasedPhotonID_Fall17_94X_V1_cff']
+        if int(os.environ["CMSSW_VERSION"].split("_")[3]) >=9:
```

Comment:
For 9_4_X, I can only find `RecoEgamma/PhotonIdentification/python/Identification/cutBasedPhotonID_Fall17_94X_V2_cff.py`
with X >= 13. I had to look in CVMFS to discover this, because the github repo only shows 9_4_X all together.

I can't find any recipes to provide this payload in earlier versions, so I would recommend two things:
    1. Change this check to `>= 13`, and do the same in CollectionProducer_cff
    2. Add a warning to setupFramework.py that if 9_4 and yet < 13, the photon VID version is only V1, and the user should consider moving to a newer release

Reply:
Also for CMSSW 8_0_X, this twiki:

https://twiki.cern.ch/twiki/bin/view/CMS/CutBasedPhotonIdentificationRun2#Recipe_for_regular_users_for_8_0

says a recipe is needed. In 8_0_X please also add a check to `setupFramework.py` similar to the other warnings at the bottom.